Hospital-System/Hospital.Email/Services/Interfaces/IEmailSender.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Hospital.Email.Base;
+
+namespace Hospital.Email.Services.Interfaces;
+
+public interface IEmailSender 
```

Comment:
This is never used! According to the Rider Search I can't find any actual usage of `IEmailSender`.

![image](https://github.com/doomer2004/Hospital-system./assets/67170413/48ff83c9-42c9-4593-ae3a-6107ba6a57be)
![image](https://github.com/doomer2004/Hospital-system./assets/67170413/adb9ffd7-1cfd-4244-8a7b-ec43d61daa03)